common/views/components/Buttons/LinkButton/LinkButton.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import {font} from '../../../../utils/classnames';
+import Icon from '../../Icon/Icon';
+
+type Props = {|
+  extraClasses?: string,
```

Comment:
Bikeshed: why aren't we using `className` in these instances and percolating down?
Legacy?

Reply:
Do you mean why don't we require one of e.g. 'primary' | 'secondary' | 'tertiary' as a prop? I wondered about that if that's what your'e asking. Potentially a more descriptive/self-documenting way to go (although we'd also need to keep the `extraClasses` for e.g. font-sizes.

Or was it something else?

Reply:
Nope - basically asking why we don't do:
```JS
<Button className={'things to add'}>Bloop</Button>